internal/model/sail.go

```diff
@@ -48,12 +56,12 @@ func (m *SailMode) Type() string {
 
 func (m *SailMode) IsEnabled() bool {
 	mode := *m
-	return mode == SailModeLocal || mode == SailModeProd
+	return mode == SailModeLocal || mode == SailModeProd || mode == SailModeStaging
 }
 
 func UnrecognizedSailModeError(v string) error {
 	return fmt.Errorf("Unrecognized sail mode: %s. Allowed values: %s", v, []SailMode{
-		SailModeDefault, SailModeDisabled, SailModeLocal, SailModeProd,
+		SailModeDefault, SailModeDisabled, SailModeLocal, SailModeProd, SailModeStaging,
```

Comment:
nit: swap Prod and Staging so they're in a reasonable order

Reply:
done